pkg/storage/memory/memory.go

```diff
@@ -106,6 +107,7 @@ func New(maxTuplesPerWrite int, maxTypesPerAuthorizationModel int) *MemoryBacken
 		authorizationModels:           make(map[string]map[string]*AuthorizationModelEntry),
 		stores:                        make(map[string]*openfgapb.Store, 0),
 		assertions:                    make(map[string][]*openfgapb.Assertion, 0),
+		currentTime:                   storage.DefaultTimeProvider{},
```

Comment:
```suggestion
		timeProvider:                   storage.DefaultTimeProvider{},
```

Reply:
Obsolete as we don't have the time provider anymore